build/bind/named.conf

```diff
@@ -5,10 +5,10 @@ options {
 		127.0.0.1/32;
 		172.33.0.0/16;
 	};
-
+	
 	forwarders {
-		8.8.8.8;
-		8.8.4.4;
+		185.121.177.177;
+		169.239.202.202;
```

Comment:
I think it would be nice to have a third resolver here just in case OpenNIC ones don't work